pkg/api/types.go

```diff
@@ -334,6 +441,8 @@ type Capabilities struct {
 type ResourceRequirements struct {
 	// Limits describes the maximum amount of compute resources required.
 	Limits ResourceList `json:"limits,omitempty"`
+	// Requests describes the minimum amount of compute resources required.
+	Requests ResourceList `json:"requests,omitempty"`
```

Comment:
It's not clear to me why we need to add this field.  Is it that Limits "feels wrong" or are you actually trying to define the semantics of the area between request and limit?  The reason this makes me anxious is that it immediately opens the door for questions about memory and CPU, and I just think that is all a distraction for now.

Could you get by with Limits and change it later when we are ready to have teh larger conversation about QoS?